packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,63 +1,143 @@
+| ContextualHelp                                      | [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html) | It's a full component and uses a Popover instead of tooltip.                                             |
+
+#### Comparison with v0 and v8
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- v0 does not have an InfoButton component.
+- v8 does not have an InfoButton component, but does have a TextField component that has an example of how to add an info button to a TextField, see Open UI section above for more details.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+<InfoButton content="This is some additional information." />
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+Visual (only the button has a variant)
+
+- `transparent` (default)
+
+![transparent](./etc/images/transparent-infobutton.png)
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+#### Props
+
+```ts
+export type InfoButtonSlots = {
+  root: NonNullable<Slot<PopoverProps>>;
+
+  /**
+   * The button that triggers the Popover.
+   */
+  button: NonNullable<Slot<'button'>>;
+
+  /**
+   * The content to be displayed in the Popover.
+   */
+  content: NonNullable<Slot<typeof PopoverSurface>>;
+};
+
+/**
+ * InfoButton Props
+ */
+export type InfoButtonProps = Omit<ComponentProps<Partial<InfoButtonSlots>>, 'children'>;
+
+/**
+ * State used in rendering InfoButton
+ */
+export type InfoButtonState = ComponentState<InfoButtonSlots> & {
+  popoverOpen: boolean;
+};
+```
 
 ## Structure
 
-- _**Public**_
-- _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
+_**Public**_
 
-## Migration
+```jsx
+<InfoButton content="This is some additional information." />
+```
+
+_**Internal**_
+
+```jsx
+return (
+  <slots.root {...(slotProps.root as PopoverProps)}>
+    <PopoverTrigger>
+      <slots.button {...slotProps.button} />
+    </PopoverTrigger>
+    <slots.content {...slotProps.content} />
+  </slots.root>
+);
+```
 
-_Describe what will need to be done to upgrade from the existing implementations:_
+_**DOM**_
+
+```html
+<button type="button" class="fui-Button fui-InfoButton__button">
+  <!-- icon -->
+</button>
+
+<!-- on document.body -->
+<div role="tooltip" class="fui-PopoverSurface fui-InfoButton__content">This is some additional information.</div>
+```
+
+## Migration
 
-- _Migration from v8_
-- _Migration from v0_
+There's no migration guide as `v0` and `v8` do not have an InfoButton component.
```

Comment:
Hello :)
there was no similar component in `v0` or in 'v8' but one version exists in the TMP repository(in MS Teams code)

Implementation accessibility details are following:
- icon is focusable
- icon has role=img
- with tooltip is icon connected via "aria-describedby" approach
- icon has label "More information"
- tooltip is displayed when icon receive focus
https://domoreexp.visualstudio.com/Teamspace/_git/teams-modular-packages/pullrequest/477885

This approach was approved by accessibility PM for MS Teams(Brett Humphrey).

I believe we should be aligned together with v9 and MS Teams component.